### PR TITLE
Week 12 Updates

### PR DESCRIPTION
**Pull request by Viswalahiri**
1. Uploaded Week 12 folder in assets
2. Updated Leaderboard
3. Changed Schedule 
4. Updated landing page to show week 12 information
5. Added Week-11 contest link in Archive.

Please review and merge accordingly @AjitZero Annaya, thank you for your time.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -95,12 +95,12 @@ <h3>We are a team of students in GITAM that aim to promote coding as being fun,
 					</div>
 				</div>
 				<div class="intro-content col-sm-10 offset-sm-1 col-lg-5 offset-lg-1 d-flex justify-content-center align-items-center flex-column">
-					<h3>You just missed Code Hackathon Week #10 ☹️</h3>
+					<h3>💻 Sign up for Code Hackathon Week #12 📈</h3>
 					<br />
-					<img style="background: rgb(26, 18, 69);" class="poster old-pic" src="assets/poster/w10.jpg" alt="Code Hackathon Week #10 Poster">
+					<img style="background: rgb(26, 18, 69);" class="poster old-pic" src="assets/poster/w12.jpg" alt="Code Hackathon Week #12 Poster">
```
Comment (ajitzero) [inline index.html]: The `old-pic` class is only used for inactive/completed event posters.

**Review by ajitzero — DISMISSED** [review]
@Viswalahiri Remove that one class and merge it.

File: index.html (hunk 1 above)
Comment (Viswalahiri) [reply index.html]: Alright, I'll remove that class then.

Comment (Viswalahiri) [reply index.html]: Done Annaya, please check.

**Review by ajitzero — APPROVED** [review]

**Pull request merged**